pydis_site/apps/content/resources/guides/pydis-guides/contributing.md

```diff
@@ -91,26 +91,57 @@ Our projects on Python Discord are open source and [available on Github](https:/
-As a token of our appreciation, those who have made significant contributions to our projects will receive a special **@Contributors** role on our server that makes them stand out from other members.
-That way, they can also serve as guides to others who are looking to start contributing to our open source projects or open source in general.
+### 1. Learn the basics of Git
+  Git is a _Version Control System_, software for carefully tracking changes to the files in a project. Git allows the same project to be worked on by people in different places. You can make changes to your local code and then distribute those changes to the other people working on the project.
 
-#### Guidelines for the @Contributors Role
+  Check out these [**resources to get started using Git**](./working-with-git/).
 
-One question we get a lot is what the requirements for the **@Contributors** role are.
-As it’s difficult to precisely quantify contributions, we’ve come up with the following guidelines for the role:
+### 2. Fork the repo
+  GitHub is a website based on the Git version control system that stores project files in the cloud. The people working on the project can use GitHub as a central place for sending their changes, getting their teammates' changes, and communicating with each other. Forking the repository that you want to work on will create a copy under your own GitHub account. You'll make your changes to this copy, then later we can bring them back to the PyDis repository.
 
-- The member has made several significant contributions to our projects.
-- The member has a positive influence in our contributors subcommunity.
+  Check out our [**guide on forking a GitHub repo**](./forking-repository/).
+### 3. Clone the repo
+  Now that you have your own fork you need to be able to make changes to the code. You can clone the repo to your local machine, commit changes to it there, then push those changes to GitHub.
 
-The role will be assigned at the discretion of the Admin Team in consultation with the Core Developers Team.
+  Check out our [**guide on cloning a GitHub repo**](./cloning-repository/).
+
+### 4. Set up the project
+  You have the source code on your local computer, now how do you actually run it? We have detailed guides on setting up the environment for each of our projects:
+
+  * [**Sir Lancebot**](./sir-lancebot/)
+
+  * [**Python Bot**](./bot/)
+
+  * [**Site**](./site/)
+
+### 5. Read our Contributing Guidelines
+  We have a few short rules that all contributors must follow. Make sure you read and follow them while working on our projects.
+
+  [**Contributing Guidelines**](./contributing-guidelines/).
+
+  As mentioned in the Contributing Guidelines, we have a simple style guide for our projects based on PEP 8. Give it a read to keep your code consistent with the rest of the codebase.
+
+  [**Style Guide**](./style-guide/)
```

Comment:
Could the links be inline? not a huge thing, but seemed a bit weird to me

Reply:
I like that it's easy to see the main links for each step at a glance. Tbh if anything I want to do the same for the creating an issue step.

Reply:
Not a big deal to me, feel free to resolve if you like the way it looks.